src/modules/fw_mode_manager/fw_mode_manager_params.c

```diff
@@ -580,6 +580,21 @@ PARAM_DEFINE_INT32(FW_LND_ABORT, 3);
  */
 PARAM_DEFINE_INT32(FW_LAUN_DETCN_ON, 0);
 
+/**
+ * Control surface launch delay
+ *
+ * Locks control surfaces during pre-launch (armed) and until this time since launch has passed.
+ * Only affects control surfaces that have corresponding flag set, and not active for runway takeoff.
```

Comment:
FWIW if there are no other docs for this, it might be worth noting the use case where you'd consider setting this here.